tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1448,6 +1458,50 @@ def test_build_layer_with_makefile_no_compatible_runtimes(self):
             "random",
         )
 
+    def test_build_layer_with_makefile_with_fake_build_architecture(self):
+        build_method = "makefile"
+        use_container = False
+        layer_identifier = "LayerWithMakefileBadArchitecture"
+
+        overrides = {
+            "LayerBuildMethod": build_method,
+            "LayerMakeContentUri": "PyLayerMake",
+            "LayerBuildArchitecture": "fake",
+        }
+        cmdlist = self.get_command_list(
+            use_container=use_container, parameter_overrides=overrides, function_identifier=layer_identifier
+        )
+
+        command_result = run_command(cmdlist, cwd=self.working_dir)
+        print(str(command_result.stderr))
```

Comment:
Let's use logging instead of print. I do think that `run_command` should log both stdout and stderr, so this call might be redundant.

Reply:
Oops, that print is leftover from my own debugging. It shouldnt be there.